webpack/devserver.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(webpackConfig) {
         chunks: false,
         chunkModules: false
       }
-    }).listen(8888, 'localhost', function(err) {
+    }).listen(8888, '0.0.0.0', function(err) {
```

Comment:
Can you please add link to issue? Also, we should ask Dan Abramov, why hot reload isn't working within VM I suppose.